bower.json

```diff
@@ -24,7 +24,7 @@
     "doc-style/**"
   ],
   "dependencies": {
-    "adapterjs": "0.14.1",
+    "adapterjs": "0.14.2",
```

Comment:
The media sources was hardcoded in the extension code previously which was updated.